### DISCUSSION TITLE
Mind hacking: understanding how our mind works · poolp.org

### DISCUSSION THREAD

**Discussion by poolpOrg** [poolpOrg/poolp.org#92] [Articles]

**Comment by bleader** [comment]
Given the look of your first part about *organic computers*, I would guess you already are familiar with the work of David Eagleman. If not I do recommand checking out the PBS serie: *The Brain* or his book *Livewire*, which I'm currently reading. It is more of a neurologist point of view, but the concept of the generic computer, sensors, and the perception of reality are there, and it definitely is a very interesting read, at least up to where I ended up until now :stuck_out_tongue:

**Comment by poolpOrg** [comment]
Thanks 😄

I'm not familiar with the work of David Eagleman, no, I used a computer analogy because I thought most of my readers are techies. I'll have a look at his work, thanks !
